setup-earthly/src/setup-earthly.js

```diff
@@ -0,0 +1,37 @@
+import * as core from '@actions/core';
+import { exec } from 'child_process'
+
+const IS_MAC = process.platform === "darwin";
+
+try {
+  let version = core.getInput("earthly-version");
```

Comment:
Should we call this just `version`?

Reply:
I was on the fence about that, I can't think of a reason we would have to install any other versions of other software here, so I think it's safe to rename this.